content/upcoming/enry.md

```diff
@@ -0,0 +1,181 @@
+---
+author: mcarmonaa
+date: 2017-07-03
+draft: true
+title: "enry: detecting your accent"
```

Comment:
As I understood it, this was an intended _pun_, since that's what Enry Iggins did.